scripts/dts/gen_defines.py

```diff
@@ -278,6 +708,47 @@ def should_write(prop):
     return True
 
 
+def prop2value(prop):
+    # Get the single macro expansion for prop.value.
+
+    if prop.type == "boolean":
+        return 1 if prop.val else 0
+    elif prop.type == "string":
```

Comment:
Could put `if` instead of `elif` after the `return`s.

Think `pylint` (and `checkpatch.pl`) warns for it, but disabled it since it's a bit case-to-case.